src/server/migrations/0.8.0-1.0.0/index.js

```diff
@@ -9,7 +9,9 @@ module.exports = {
 	fromVersion: '0.8.0',
 	toVersion: '1.0.0',
 	up: async db => {
-		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/create_cik_table.sql'));
-		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/create_compressed_reading_views.sql'));
+		await db.none(sqlFile('../migrations/0.8.0-1.0.0/sql/cik/create_cik_table.sql'));
```

Comment:
I think the repository `src/server/migrations/0.8.0-1.0.0/sql/cij` has been accidentally mislabeled. We should change it into `cik` for this line to work.

Reply:
Yes. I have renamed the directory to cik.